test-suite/unit-tests/ide/lex_tests.ml

```diff
@@ -0,0 +1,50 @@
+open Utest
```

Comment:
Do we have a mechanism to skip ide tests when gtk isn't available?
Alternatively, coq_lex depends on gtk to pass `Tags.Script.comment` and `Tags.Script.sentence` to the closure it takes as argument, maybe we could refactor to remove the dependency and move coq_lex to ide_common.

Reply:
Is there a CI setup running test-suite and not having lablgtk installed?

Reply:
Apparently no, so this PR seems OK. But I see your point, and my preferred solution would be making the lexer not depend on GTK ;-)